drivers/led_strip/apa102.c

```diff
@@ -21,7 +21,17 @@ static int apa102_update(const struct device *dev, void *buf, size_t size)
 {
 	const struct apa102_config *config = dev->config;
 	static const uint8_t zeros[] = { 0, 0, 0, 0 };
```

Comment:
And is that OK to have only 32 zeros if there are more than 64 LEDs ?

Reply:
yes

Reply:
OK ! :)